spec/services/figgy_event_processor_spec.rb

```diff
@@ -88,15 +88,15 @@
       end
     end
     context "when it's no longer accessible" do
-      it "marks it as non-public" do
+      it "deletes it from solr" do
```

Comment:
Add an assertion that it's still in the db?

Reply:
Sounds good - done.